01-shapefile-attributes.Rmd

```diff
@@ -72,19 +73,20 @@ Crop and extract raster values in R</a>
 </div>
 
 NOTE: The data used in this tutorial were collected at Harvard Forest which is
-a the National Ecological Observatory Network field site <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are proxy data for what will be
-available for 30 years from the NEON flux tower [from the NEON data portal](http://data.neoninc.org/ "NEON data").
+a the National Ecological Observatory Network field site <a 
+href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
+More about the NEON Harvard Forest field site</a>. These data are proxy data for
+what will be available for 30 years from the NEON flux tower [from the NEON data portal](http://data.neoninc.org/ "NEON data").
 {: .notice}
  
 ##Notes about R Libraries
 
-To work with vector data in R, we can use the `rgdal` library. We will load the `raster`
-library to work with rasters. The `raster` library also allows us to explore metadata
-using similar commands with both rasters and vectors.
- 
-##Querying shapefile attributes
+To work with vector data in R, we can use the `rgdal` library. We will load the 
+`raster` library to work with rasters. The `raster` library also allows us to 
+explore metadata using similar commands with both rasters and vectors.
 
+##Setting up the workspace
+If you completed Lesson 00 {{site.baseurl }} R/open-shapefiles-in-R/ you can skip this code chunk.
```

Comment:
the syntax should be as follows:
   [{{site.baseurl }} R/open-shapefiles-in-R/](Lesson 00 - Introduction to shapefiles)